src/wallet/rpcwallet.cpp

```diff
@@ -2448,18 +2448,17 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "\nResult (output indices for only one value pool will be present):\n"
             "[                             (array of json object)\n"
             "  {\n"
-            "    \"txid\" : \"txid\",          (string) the transaction id \n"
-            "    \"pool\" : \"sprout|sapling|orchard\", (string) The shielded value pool\n"
-            "    \"type\" : \"sprout|sapling|orchard\", (string) The shielded value pool (DEPRECATED legacy attribute)\n"
-            "    \"jsindex\" (sprout) : n,       (numeric) the joinsplit index\n"
-            "    \"jsoutindex\" (sprout) : n,       (numeric) the output index of the joinsplit\n"
-            "    \"outindex\" (transparent, sapling, orchard) : n,       (numeric) the Output or Action index\n"
-            "    \"confirmations\" : n,       (numeric) the number of confirmations\n"
-            "    \"spendable\" : true|false,  (boolean) true if note can be spent by wallet, false if address is watchonly\n"
-            "    \"address\" : \"address\",    (string) the shielded address\n"
-            "    \"amount\": xxxxx,          (numeric) the amount of value in the note\n"
-            "    \"memo\": xxxxx,            (string) hexadecimal string representation of memo field\n"
-            "    \"change\": true|false,     (boolean) true if the address that received the note is also one of the sending addresses\n"
+            "    \"txid\" : \"txid\",                   (string) the transaction id \n"
+            "    \"pool\" : \"sprout|sapling|orchard\",   (string) The shielded value pool\n"
```

Comment:
I confirmed that the only release "type" was present in was 4.7.0-rc1, and so it does not need to be retained for compatibility (since there is no commitment to API compatibility with RCs).